cloudflare_exporter/collector.py

```diff
@@ -86,7 +86,22 @@ def collect(self):
 
 def _get_cloudflare_analytics(token):
     cloudflare = CloudFlare.CloudFlare(debug=False, token=token)
-    for zone in cloudflare.zones.get():
+    try:
+        zones = cloudflare.zones.get()
+    except CloudFlare.exceptions.CloudFlareAPIError as e:
+        exit('/zones.get %d %s - api call failed' % (e, e))
+    except Exception as e:
+        exit('/zones.get - %s - api call failed' % (e))
```

Comment:
This is a bad pattern (too often used), it should fail the whole process.

Reply:
I rewrite this part

Reply:
If a user ( token) don't have acces to a cloudflare zone ( like "criteo.com" ), we don't want to crash, and export others zones